packages/renderer/src/lib/image/ImageColumnName.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+import { router } from 'tinro';
+import type { ImageInfoUI } from './ImageInfoUI';
+
+export let object: ImageInfoUI;
+
+function openDetailsImage(image: ImageInfoUI) {
+  router.goto(`/images/${image.id}/${image.engineId}/${image.base64RepoTag}/summary`);
+}
+</script>
+
+<!-- svelte-ignore a11y-click-events-have-key-events -->
+<!-- svelte-ignore a11y-no-static-element-interactions -->
```

Comment:
should we use `<button>` below ? AFAIK it makes the expected behaviour rather than applying on:click on a div